lib/dragonfly_audio/processors/tag.rb

```diff
@@ -0,0 +1,29 @@
+require 'taglib'
+
+# IMPORTANT: See http://robinst.github.io/taglib-ruby/
+
+module DragonflyAudio
+  module Processors
+    class Tag
```

Comment:
I guess same here, would it be possible to use the constants from AudioProperties classs?